packages/server/tests/previewnetAcceptance.env

```diff
@@ -0,0 +1,42 @@
+HEDERA_NETWORK=previewnet
+CHAIN_ID=0x129
+MIRROR_NODE_URL=https://previewnet.mirrornode.hedera.com
+LOCAL_NODE=false
+MIRROR_NODE_URL_WEB3=https://previewnet.mirrornode.hedera.com
+SERVER_PORT=7546
+RELAY_ENDPOINT='http://127.0.0.1:7546'
+# cypress metamask config
```

Comment:
Most if not all the values below are not need in this and the  testnet file.
Can we simplify the files to only the ones that need to override the defaults.